### PR TITLE
#167190440 Attach authors to articles created by them

### PR DESCRIPTION
**Pull request by chiboycalix**
#### What are the relevant pivotal tracker stories?
[#167190440](https://www.pivotaltracker.com/story/show/167190440)
#### Screenshots (if appropriate)

Screenshot 2019-08-05 at 10 20 21 AM

### CODE REVIEW

**Review by encodedBicoding — APPROVED** [review]

**Review by encodedBicoding — APPROVED** [review]
LGTM

File: server/db/migrations/20190730213651-create-article.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ module.exports = {
       type: Sequelize.STRING
     },
     articleBody: {
-      type: Sequelize.STRING
+      type: Sequelize.TEXT
```
Comment (chuxmykel) [inline server/db/migrations/20190730213651-create-article.js]: Has this been addressed?

File: server/db/models/article.js [hunk 2]
```diff
@@ -6,15 +6,14 @@ module.exports = (sequelize, DataTypes) => {
     'Article',
     {
       title: DataTypes.STRING,
-      articleBody: DataTypes.STRING,
+      articleBody: DataTypes.TEXT,
```
Comment (chuxmykel) [inline server/db/models/article.js]: This too

File: server/db/migrations/20190730213651-create-article.js (hunk 1 above)
Comment (chiboycalix) [reply server/db/migrations/20190730213651-create-article.js]: Yes Please

File: server/db/models/article.js (hunk 2 above)
Comment (chiboycalix) [reply server/db/models/article.js]: Yes It has

**Review by chuxmykel — APPROVED** [review]
Nice. Looks good.

**Review by Hadeneekeh — APPROVED** [review]
LGTM

**Review by Lundii — APPROVED** [review]
Nice work. LGTM

**Review by topseySuave — APPROVED** [review]

**Pull request merged**